debian/libmariadb-dev.install

```diff
@@ -12,10 +12,10 @@ usr/include/mariadb/mariadb_dyncol.h
 usr/include/mariadb/mariadb_rpl.h
 usr/include/mariadb/mariadb_stmt.h
 usr/include/mariadb/mariadb_version.h
+usr/include/mariadb/my_alloca.h
 usr/include/mariadb/my_config.h
 usr/include/mariadb/my_global.h
 usr/include/mariadb/my_sys.h
-usr/include/mariadb/my_alloca.h
```

Comment:
This does not belong here?

Reply:
It was already here, I just ran `wrap-and-sort -v` to have the file listed alphabetically.